vllm/worker/model_runner.py

```diff
@@ -156,6 +157,9 @@ def __init__(
         # (max batch size to capture, max context len to capture / block size).
         self.graph_block_tables: torch.Tensor  # Set after initial profiling.
 
+        # Set if the backend is flashinfer.
+        self.workspace_buffer = None
```

Comment:
```suggestion
        self.flash_infer_workspace_buffer: <add type>
```

Also you don't need to set `= None` if you know it is not used unless it is lazily initialized. See https://github.com/vllm-project/vllm/blob/4ea1f9678dd93f02424ab3de2149f83a490e6c6f/vllm/worker/model_runner.py#L149